tos.html

```diff
@@ -1,4 +1,4 @@
-<p>These Terms of Service (&quot;Terms&quot;) govern your use of any Codidact community hosted at <a href="https://codidact.com/">https://codidact.com/</a> or any subdomain thereof (the &quot;Service&quot;). These Terms apply to all uses of the Service, whether you sign up for an account or not. Your use of the Service constitutes your acceptance of these Terms.</p>
+<p>These Terms of Service (&quot;Terms&quot;) govern your use of any community hosted by The Codidact Foundation, including <a href="https://codidact.com/">https://codidact.com/</a>, <a href="https://codidact.org">https://codidact.org</a>, and subdomains thereof (the &quot;Service&quot;). These Terms apply to all uses of the Service, whether you sign up for an account or not. Your use of the Service constitutes your acceptance of these Terms.</p>
```

Comment:
We lost "They are an agreement between you and Codidact C.I.C. ("We" or "Us"). " from your original PR in the merge.